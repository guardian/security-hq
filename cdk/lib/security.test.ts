// This file was autogenerated using @guardian/cdk-cli

import '@aws-cdk/assert/jest';
import { SynthUtils } from '@aws-cdk/assert';
import { App } from '@aws-cdk/core';
import { Security } from './security';

describe('The Security stack', () => {
  it('matches the snapshot', () => {
    const app = new App();
    const stack = new Security(app, 'security', { stack: 'security' });
    expect(SynthUtils.toCloudFormation(stack)).toMatchSnapshot();
  });
});
