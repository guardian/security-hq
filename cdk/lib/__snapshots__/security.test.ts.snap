// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Security stack matches the snapshot 1`] = `
Object {
  "Parameters": Object {
    "Stage": Object {
      "AllowedValues": Array [
        "CODE",
        "PROD",
      ],
      "Default": "CODE",
      "Description": "Stage name",
      "Type": "String",
    },
  },
}
`;
